roofit/roofitcore/inc/RooFit/TestStatistics/LikelihoodGradientWrapper.h

```diff
@@ -37,12 +38,22 @@ class LikelihoodGradientWrapper {
    LikelihoodGradientWrapper(std::shared_ptr<RooAbsL> likelihood,
                              std::shared_ptr<WrapperCalculationCleanFlags> calculation_is_clean, std::size_t N_dim,
                              RooMinimizer *minimizer);
+   LikelihoodGradientWrapper(std::shared_ptr<RooAbsL> likelihood,
+                             std::shared_ptr<WrapperCalculationCleanFlags> calculation_is_clean, std::size_t N_dim,
+                             RooMinimizer *minimizer,
+                             std::shared_ptr<std::vector<ROOT::Math::KahanSum<double>>> offsets,
+                             std::shared_ptr<std::vector<ROOT::Math::KahanSum<double>>> offsets_save);
```

Comment:
```suggestion
   LikelihoodGradientWrapper(std::shared_ptr<RooAbsL> likelihood,
                             std::shared_ptr<WrapperCalculationCleanFlags> calculation_is_clean, std::size_t N_dim,
                             RooMinimizer *minimizer,
                             std::shared_ptr<std::vector<ROOT::Math::KahanSum<double>>> offsets=nullptr,
                             std::shared_ptr<std::vector<ROOT::Math::KahanSum<double>>> offsets_save=nullptr);
```
I would suggest default arguments here, so that you can delete the original overload. Having separate definitions poses a risk that they go out of sync.

Also, can this constructor be made `private` to make it clear that it's not meant to be used directly? All instantiation should happen via `create()`, right?

Reply:
Done and done.